datumaro/components/dataset.py

```diff
@@ -823,7 +838,7 @@ def import_from(cls, path: str, format: str = None, env: Environment = None,
                 'url': path, 'format': format, 'options': kwargs
             }]
         else:
-            raise DatumaroError("Unknown source format '%s'. To make it "
+            raise UnknownFormatError("Unknown source format '%s'. To make it "
                 "available, add the corresponding Extractor implementation "
                 "to the environment" % format)
```

Comment:
Done

Reply:
OK, but there are also other cases where the message is specified in the `raise` statement instead of the class (I don't want to point them out manually - search for `raise`s).

Reply:
I adjusted other uses as well

Reply:
Not all of them. There's a `NoMatchingFormatsError` in this file, plus `DatasetNotFoundError` in `extractor.py`.

Reply:
Done